routes/userRoutes.js

```diff
@@ -28,4 +29,8 @@ router.route('/user')
     emailExistsValidator,
     updateUser);
 
+router.route('/authors')
+  .all(verifyToken)
```

Comment:
Since there is just one http verb (`get`) used here, there's no need to use the `all` method. The `get` should accept `verifyToken` as the first parameter, and its controller as the second parameter.

Reply:
ok. I thought it will be ok to have it incase of future changes to author route. Refactoring it asap

Reply:
I've done that